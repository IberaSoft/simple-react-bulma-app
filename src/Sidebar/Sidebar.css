aside.menu {
    padding-top: 3rem;
  }

aside.menu .menu-label {
    padding-left: 10px;
    font-weight: 700;
}