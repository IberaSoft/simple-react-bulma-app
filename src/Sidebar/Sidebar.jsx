import React from 'react';
import Navigation from './../Navigation/Navigation';
//css
import './Sidebar.css';

const Sidebar = () => {
    return (
        <section className="sidebar">
            <Navigation className="navigation-vertical" />
        </section>
    );
}

export default Sidebar;