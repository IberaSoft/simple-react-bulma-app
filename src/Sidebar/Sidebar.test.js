import React from 'react'
import Sidebar from './Sidebar'
import ShallowRenderer from 'react-test-renderer/shallow';

describe('Sidebar component', () => {
    it('Should match the snapshot', () => {
        const renderer = new ShallowRenderer();
        const result = renderer.render(<Sidebar />);
        expect(result).toMatchSnapshot();
    });
})