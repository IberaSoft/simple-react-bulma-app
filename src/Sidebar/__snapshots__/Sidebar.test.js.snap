// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component Should match the snapshot 1`] = `
<aside
  className="menu"
>
  <p
    className="menu-label"
  >
    Sidebar
  </p>
  <Navigation
    activeIndex={undefined}
    changeIndex={undefined}
    menuItems={undefined}
  />
</aside>
`;
